runtime/jit_vm/ctsupport.cpp

```diff
@@ -720,4 +720,64 @@ jitGetDeclaringClassOfROMField(J9VMThread *vmStruct, J9Class *clazz, J9ROMFieldS
 	return currentClass;
 }
 
+void
+allMethodFromSignatureEndDo(J9MethodFromSignatureWalkState *state)
+{
+	J9JavaVM *vm = state->classLoaderWalkState.vm;
+	vm->internalVMFunctions->allClassLoadersEndDo(&state->classLoaderWalkState);
+}
+
+J9Method*
+allMethodFromSignatureNextDo(J9MethodFromSignatureWalkState *state)
+{
+	UDATA result = 0;
+
+	J9JavaVM *vm = state->classLoaderWalkState.vm;
+	J9ClassLoader *classLoader = vm->internalVMFunctions->allClassLoadersNextDo(&state->classLoaderWalkState);
+	while (NULL != classLoader) {
+		J9Class *clazz = vm->internalVMFunctions->internalFindClassUTF8(state->vmThread, state->className, state->classNameLength, classLoader, J9_FINDCLASS_FLAG_EXISTING_ONLY);
+		if (NULL != clazz) {
+			result = vm->internalVMFunctions->javaLookupMethod(state->vmThread, clazz, reinterpret_cast<J9ROMNameAndSignature*>(&state->nameAndSig), NULL, J9_LOOK_JNI | J9_LOOK_NO_JAVA);
+			if (0 != result) {
+				break;
+			}
+		}
+
+		classLoader = vm->internalVMFunctions->allClassLoadersNextDo(&state->classLoaderWalkState);
+	}
+
+	return reinterpret_cast<J9Method*>(result);
+}
+
+J9Method*
+allMethodFromSignatureStartDo(J9MethodFromSignatureWalkState *state, J9JavaVM *vm, UDATA flags, U_8 *className, U_16 classNameLength, U_8 *methodName, U_16 methodNameLength, U_8 *methodSig, U_16 methodSigLength)
+{
+	state->className = className;
+	state->classNameLength = classNameLength;
+	state->nameAndSig.name = reinterpret_cast<char*>(methodName);
```

Comment:
The name parameters should be of type `const U8 *` and the `const` should not be cast away here.

Reply:
See above.